style.css

```diff
@@ -502,3 +500,269 @@ textarea::placeholder {
+  }
+
+  .nav ul {
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    list-style: none;
+    padding: 0 1rem 0 1rem;
+    width: 16.188rem;
+    margin-right: 8.75rem;
+  }
+
+  .nav li {
+    font-family: "poppins", sans-serif;
+    color: #334563;
+    font-weight: 500;
+    font-size: 15px;
+    padding: 0 0.75rem;
+  }
+
+  .hamburger {
+    /* color: red; */
+    visibility: hidden;
+  }
+
+  .skill-section ul {
+    /* display: flex;  1 */
+    padding-left: 20px;
+  }
+
+  /* .skill-dropdown-content li {
+    height: 66px;
+    background-color: #f7f7f9;
+    width: 100%;
+    padding: 8px 16px;
+    display: flex;
+    border-radius: 8px;
+  } */
+
+  .skill-section ul li {
+    /* flex-direction: column;2 */
+    height: 100px;
+
+    /* margin-right: 12px;3 */
+  }
+
+  .skill-dropdown-content {
+    display: flex;
+    flex-direction: row;
+    justify-content: flex-start;
+    align-items: center;
+    padding: 12px 32px;
+    gap: 12px;
+  }
+
+  /* .skill-dropdown-content li {
+    height: 120px;
+    width: 122px;
+    background-color: #f7f7f9;
+    padding: 8px 0;
+    display: flex;
+    flex-direction: column;
+    border-radius: 8px;
+    text-align: center;
+  } */
+
+  .nav li:enabled {
+    color: #6070ff;
+    background: #fff;
+  }
+
+  .nav li:hover {
+    color: #fff;
+    background: #6070ff;
+  }
+
+  .nav li:active {
+    color: #fff;
+    background: #2230d2;
+  }
+
+  .nav li:disabled {
+    color: #5e6c84;
+    background: #fff;
+  }
+
+  .content {
+    background-image: url(./images/HeaderD.png);
+    background-position: right;
+    background-size: contain;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    height: 650px;
+  }
+
+  .sec-br {
+    display: none;
+  }
+
+  .content div {
+    width: 65%;
+  }
+
+  .intro {
+    padding: 0;
+    margin-top: 0;
+    margin-bottom: 0;
+    height: fit-content;
+    display: flex;
+    align-items: flex-start;
+  }
+
+  .card {
+    display: flex;
+    height: fit-content;
+    width: 85%;
+    align-items: center;
+    justify-content: center;
+    flex-direction: row;
+    margin-left: 8.875rem;
+    margin-right: 8.875rem;
+  }
+
+  .works-section > :nth-child(2n) {
+    flex-direction: row-reverse;
+  }
+
+  .card > * {
+    width: 50%;
+    height: 20.813rem;
+    padding-left: 1rem;
+  }
+
+  .card-description {
+    font-weight: 400;
+    font-size: 13px;
+    width: 95%;
+    height: fit-content;
+    margin: 0;
+  }
+
+  .card-info {
+    height: fit-content;
+  }
+
+  .card-image {
+    /* width: 50%; */
+    height: 100%;
+  }
+
+  .primary-button {
+    flex: none;
+    margin-top: 0;
+  }
+
+  .card-tags {
+    display: flex;
+  }
+
+  .card-title {
+    /* margin-top: 5px; */
+    margin-bottom: 0;
+  }
+
+  .card-title br {
+    display: none;
+  }
+
+  .about-me-section {
+    display: flex;
+    flex-direction: row;
+    padding-top: 100px;
+    height: 450px;
+  }
```

Comment:
- [ ] As mentioned above let's also fix the about me section to match the figma template as close as possible please see the current design 
![image](https://user-images.githubusercontent.com/97521557/208238075-f1367d35-a9db-4e99-87ec-bb06f379e852.png)

How it should be according to the figma template 
![image](https://user-images.githubusercontent.com/97521557/208238161-14121a26-58c8-4492-8197-561efaff12c5.png)